sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/GotoStatement.vb

```diff
@@ -0,0 +1,28 @@
+﻿Imports System
+
+Namespace Tests.Diagnostics
+
+    Public Class GotoStatement
+
+        Sub GotoStatement(condition As Boolean)
+
+            If stubid Then
+                GoTo Label 'Noncompliant
```

Comment:
Please add one message verification as well using `{{message}}` comment (see tests for` GotoStatement.cs`). I forgot this one before.